components/progress-bar/blender/recipe.js

```diff
@@ -0,0 +1,34 @@
+import { GEL } from '@westpac/core';
+import React from 'react';
+
+import { ProgressBar } from '@westpac/progress-bar';
+
+export function AllStyles({ brand }) {
+	return (
+		<GEL brand={brand}>
+			<ProgressBar value={20} />
+			<ProgressBar value={20} look="skinny" />
```

Comment:
I'd also include `<ProgressBar value={0} />` and `<ProgressBar value={100} />` just to be sure